README.md

```diff
@@ -49,10 +49,7 @@ bin/magento cache:clean;
 
 ## Check after installation
 
-1. After installation check the results of the cron operations. Cron job name `storekeeper_storekeeper_cron_orders`
-   has to be added to `cron_schedule` DB table. Check `var/log/cron.log` file as well.
-
-2. Check `queue_message` DB table. Topic `storekeeper.queue.events` with correct JSON body has to be added in it.
+1. Check `queue_message` DB table. Topic `storekeeper.queue.events` with correct JSON body has to be added in it.
```

Comment:
also needs to be changed https://github.com/storekeeper-company/magento2-plugin?tab=readme-ov-file#queue